hamocc/mo_ocprod.F90

```diff
@@ -137,6 +137,7 @@ subroutine ocprod(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,omask,ptho,pi_ph,psao,pp
     real :: wpocd,wcald,wopald,wdustd,dagg
     real :: wcal,wdust,wopal,wpoc
     real :: o2lim ! O2 limitation of ammonification (POC remin)
+    real, parameter :: tiny_num = 1e-25
```

Comment:
Do you need to define this parameter? If you just need a small number, there is the intrinsic function "tiny" for which tiny(0.0) will return the smallest positive real number.
See e.g. https://fortran-lang.org/en/learn/intrinsics/model/#tiny

Reply:
Good question, thanks - I exchanged the value by the epsilon function now.

Reply:
```suggestion
              nh4uptfrac = anh4lim/(nlim+epsilon(1.))
```
Why not replace the `tiny_num` parameter? Now you have to look in the variable definition list to see that `tiny_num = epsilon(1.)`

Reply:
That didn't come out as I intended. Point is, I'm not sure you need a `tiny_num` variable at all.

Reply:
I'll/should go through the code again exchanging those numbers, which I'll do.